src/main/java/net/minecraftforge/fmp/ForgeMultipart.java

```diff
@@ -0,0 +1,93 @@
+package net.minecraftforge.fmp;
+
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.Mod.EventHandler;
+import net.minecraftforge.fml.common.SidedProxy;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLMissingMappingsEvent;
+import net.minecraftforge.fml.common.event.FMLMissingMappingsEvent.MissingMapping;
+import net.minecraftforge.fml.common.event.FMLPostInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import net.minecraftforge.fml.common.registry.GameRegistry.Type;
+import net.minecraftforge.fmp.block.BlockMultipartContainer;
+import net.minecraftforge.fmp.block.TileCoverable;
+import net.minecraftforge.fmp.block.TileMultipartContainer;
+import net.minecraftforge.fmp.network.MultipartNetworkHandler;
+import net.minecraftforge.fmp.util.MCMPEventHandler;
+
+@Mod(modid = ForgeMultipart.MODID, name = ForgeMultipart.NAME, version = ForgeMultipart.VERSION)
```

Comment:
Usually things like these hooks go into FML sided handlers directly. Lex/cpw should know more.